example/src/main/java/io/netty/example/http/helloworld/HttpHelloWorldServerHandler.java

```diff
@@ -52,11 +50,12 @@ public void channelRead0(ChannelHandlerContext ctx, HttpObject msg) {
             response.headers().set(CONTENT_TYPE, "text/plain");
             response.headers().setInt(CONTENT_LENGTH, response.content().readableBytes());
 
-            if (!keepAlive) {
-                ctx.write(response).addListener(ChannelFutureListener.CLOSE);
-            } else {
-                response.headers().set(CONNECTION, KEEP_ALIVE);
+            if (keepAlive) {
```

Comment:
@kachayev one question... isn't this assuming that we are talking `HTTP/1.1` ? I mean for 1.0 we would need to add the `keep-alive` imho

Reply:
Yeah, probably you're correct: we're responding with 1.1 but any 1.0 client would not be able to process this correctly.

Reply:
Mind fixing it ?

Reply:
Sure, will do.

Reply:
@normanmaurer Done! `HttpStaticFileServerHandler` was not that simple to rework because of exception catch branch. I also updated `HttpUploadServerHandler` to use `HttpUtils`. PTAL.